src/qureg_init.cpp

```diff
@@ -57,7 +58,7 @@ void QubitRegister<Type>::Resize(std::size_t new_num_amplitudes)
   log2_nprocs = iqs::ilog2(nprocs);
 
   // FIXME GG: I believe this limits the use of "resize" to adding a single qubit
-  if(GlobalSize()) assert(GlobalSize() * 2UL == new_num_amplitudes);
+  // if(GlobalSize()) assert(GlobalSize() * 2UL == new_num_amplitudes);
```

Comment:
Resize() will be deprecated in the next release in favor of a more descriptive method that clarifies how the state is expanded or reduced in size.